distribution/docs/src/main/resources/_contents/_catalog-frameworks/standard-catalog-framework-contents.adoc

```diff
@@ -0,0 +1,152 @@
+
+==== Standard Catalog Framework
+
+The Standard Catalog Framework provides the reference implementation of a Catalog Framework that implements all requirements of the ${ddf-catalog} API. 
+`CatalogFrameworkImpl` is the implementation of the ${branding} Standard Catalog Framework.
+
+The Standard Catalog Framework is the core class of ${branding}.
+It provides the methods for query, create, update, delete, and resource retrieval (QCRUD) operations on the `Sources`.
+By contrast, the Fanout Catalog Framework only allows for query and resource retrieval operations, no catalog modifications, and all queries are enterprise-wide.
+
+Use this framework if:
+
+* access to a catalog provider to create, update, and delete catalog entries are required.
+* queries to specific sites are required.
+* queries to only the local provider are required.
+
+It is possible to have only remote Sources configured with no local `CatalogProvider` configured and be able to execute queries to specific remote sources by specifying the site name(s) in the query request.
+
+The Standard Catalog Framework also maintains a list of `ResourceReaders` for resource retrieval operations.
+A resource reader is matched to the scheme (i.e., protocol, such as `file://`) in the URI of the resource specified in the request to be retrieved.
+
+Site information about the catalog provider and/or any federated source(s) can be retrieved using the Standard Catalog Framework.
+Site information includes the source's name, version, availability, and the list of unique content types currently stored in the source (e.g., NITF).
+If no local catalog provider is configured, the site information returned includes site info for the catalog framework with no content types included.
+
+===== Installing the Standard Catalog Framework
+
+The Standard Catalog Framework is bundled as the `catalog-core-standardframework` feature and can be installed and uninstalled using the normal processes described in Configuration.
+
+When this feature is installed, the Catalog Fanout Framework App feature `catalog-core-fanoutframework` should be uninstalled, as both catalog frameworks should not be installed simultaneously.
```

Comment:
The `catalog-core-fanoutframework` feature does not exist. There is currently only one implemented framework.